qemu-test.sh

```diff
@@ -217,50 +220,25 @@ process_args() {
     fi
 
     case ${VM_TYPE} in
-        "td-1.0")
+        "td")
             cpu_tsc=$(grep 'cpu MHz' /proc/cpuinfo | head -1 | awk -F: '{print $2/1024}')
             if (( $(echo "$cpu_tsc < 1" |bc -l) )); then
                 PARAM_CPU+=",tsc-freq=1000000000"
             fi
             # Note: "pic=no" could only be used in TD mode but not for non-TD mode
-            PARAM_MACHINE+=",kernel_irqchip=split,memory-encryption=tdx"
+            PARAM_MACHINE+=",kernel_irqchip=split,confidential-guest-support=tdx"
             QEMU_CMD+=" -bios ${OVMF}"
-            QEMU_CMD+=" -object tdx-guest,sept-ve-disable=on,id=tdx"
-            if [[ ${QUOTE_TYPE} == "tdvmcall" ]]; then
-                QEMU_CMD+=",quote-generation-service=vsock:2:4050"
-            fi
-            if [[ -n ${MROWNERCONFIG} ]]; then
-                QEMU_CMD+=",mrownerconfig=${MROWNERCONFIG}"
-            fi
-            if [[ ${DEBUG} == true ]]; then
-                QEMU_CMD+=",debug=on"
-            fi
-            ;;
-    	"td-1.5")
-            cpu_tsc=$(grep 'cpu MHz' /proc/cpuinfo | head -1 | awk -F: '{print $2/1024}')
-            if (( $(echo "$cpu_tsc < 1" |bc -l) )); then
-                PARAM_CPU+=",tsc-freq=1000000000"
-            fi
-            # Note: "pic=no" could only be used in TD mode but not for non-TD mode
-            PARAM_MACHINE+=",kernel_irqchip=split,confidential-guest-support=tdx,memory-backend=ram1"
-            QEMU_CMD+=" -bios ${OVMF}"
-            QEMU_CMD+=" -object tdx-guest,sept-ve-disable=on,id=tdx"
+
+            # Get quote using configfs-tsm
             if [[ ${QUOTE_TYPE} == "tdvmcall" ]]; then
-                QEMU_CMD+=",quote-generation-service=vsock:2:4050"
+                QEMU_CMD+=" -object '{\"qom-type\":\"tdx-guest\",\"id\":\"tdx\",\"quote-generation-socket\":{\"type\":\"vsock\",\"cid\":\"1\",\"port\":\"4050\"}}'"
```

Comment:
Please try command without `tdvmcall`, maybe we should add `else`:
```Shell
else
    QEMU_CMD+=" -object tdx-guest,id=tdx"
```

Reply:
That's correct. Updated.